src/coreclr/nativeaot/Runtime/threadstore.cpp

```diff
@@ -82,8 +82,11 @@ ThreadStore * ThreadStore::Create(RuntimeInstance * pRuntimeInstance)
     if (NULL == pNewThreadStore)
         return NULL;
 
+    // TODO-LLVM: will wasm threads help with this?
```

Comment:
```suggestion
```
(I doubt WASM will ever be capable of using this suspension technique)

Reply:
removed.